### DISCUSSION TITLE
Build a Python Package Step by Step | SF-Zhou's Blog

### DISCUSSION THREAD

**Discussion by SF-Zhou** [SF-Zhou/sf-zhou.github.io#181] [Announcements]
https://sf-zhou.github.io/programming/python_package_dingtalk_robot.html

**Comment by SF-Zhou** [comment]
本文 `3. 构建 Python 包` 节中，在 `setup.py` 中使用 `import dingtalk_robot` 会导致部分用户安装失败（想想为什么）。GitHub 中的增加的 commit [step 7](https://github.com/SF-Zhou/DingtalkRobot/commit/4ef8eccc1edecb224c2e30287d2020325e199c78) 修复了该问题。

**Comment by WingsGo** [comment]
是因为import dingtalk_robot时，会先import request对于没有安装request库的用户，便会导致安装失败吗
